src/sentry/static/sentry/app/components/acl/featureDisabled.tsx

```diff
@@ -109,6 +115,18 @@ class FeatureDisabled extends React.Component<Props, State> {
                 }
               )}
             </p>
+            <Clipboard hideUnsupported value={installText(features, featureName)}>
+              <Button
+                borderless
+                size="xsmall"
+                onClick={e => {
+                  e.stopPropagation();
+                  e.preventDefault();
+                }}
+              >
+                <StyledIconCopy /> Copy to Clipboard
```

Comment:
```suggestion
                <StyledIconCopy /> {t('Copy to Clipboard')}
```
Should use translation functions.